### ISSUE TITLE
About the issue of sponge permission

### ISSUE THREAD

**Issue by mcqwy** [The-Fireplace-Minecraft-Mods/Clans#153]
Clans Version : 1.5.0
When I try to use the opclan command
It reminds me that I have no authority
This is the permission I added to the Sponge server
The opclan command can be executed at the console.
![image](https://user-images.githubusercontent.com/38318368/71842944-c64cc300-30fd-11ea-84b0-a9406db21656.png)
I try to find problems in the source code
Unfortunately, it took more than five hours
Didn't find out what the problem was.
So far I don't understand
Is it caused by my permission adding error or version problem

**Comment by The-Fireplace** [comment]
Try also granting `clans.command.opclan`

**Comment by mcqwy** [comment]
![image](https://user-images.githubusercontent.com/38318368/71870878-53216c00-3152-11ea-91c1-04e858453e59.png)
It still doesn't work
I tried it yesterday
Forge version forge-1.12.2-14.23.5.2838-universal
Sponge version spongforge-1.12.2-2838-7.1.7
LuckPerms-Sponge-5.0.39

Only after the opclan command adds permissions. It cannot be used
After adding permissions to the clam command and raid command, you can use the
Unfortunately, it reminds me that I still don't have permission

**Comment by The-Fireplace** [comment]
Don't do `clans.command.clans.opclan`, `clans.command.clans.opclan.*`, or `clans.command.clans.*`.
`command.clans.opclan` should be `command.clans.opclan.*` and you should add `clans.command.opclan`. Let me know if that fixes it.

**Comment by mcqwy** [comment]
LuckPerms plug-in works well
It ignores incorrect permissions when they are added
But once the corresponding permission is added, it will be effective immediately
For example:
After I add `command.clans.clan.claim` permission, I can use the `/clan claim` instruction


It doesn't work `command.clans.opclan` and `command.clans.opclan.*`

**Comment by The-Fireplace** [comment]
> LuckPerms plug-in works well
It ignores incorrect permissions when they are added

I know, I have used LuckPerms before. I know it ignores incorrect permissions, I was just telling you there is absolutely no reason those should have been added, because they won't do anything.

> It doesn't work `command.clans.opclan` and `command.clans.opclan.*`

The first should be `clans.command.opclan` not `command.clans.opclan`. The reason being that `command.clans.opclan.*` is the permission I added in Clans, but `clans.command.opclan` is one that I've seen other mods/plugins add as a generic permission to be able to use the command, even though I have permissions set up for it.
